ship/src/main/scala/ch/epfl/bluebrain/nexus/ship/Main.scala

```diff
@@ -55,9 +70,18 @@ object Main
       _      <- logger.info(config.root().render())
     } yield ()
 
+  private def s3client(config: ShipConfig) = Interpreter[IO]
```

Comment:
Could be done later but we could reuse the S3 client in storage ?

Reply:
I made the change but it's not exactly the same listing method, because in the ship I don't want to list everything in the bucket, but everything with a specific prefix ("folder")